packages/trader/src/Modules/Trading/Components/Form/Purchase/__tests__/contract-info.spec.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { TProposalTypeInfo } from 'Types';
+import { getLocalizedBasis, TURBOS } from '@deriv/shared';
+import ContractInfo from '../contract-info';
+
+const test_message = 'Some test message';
+const value_movement = 'ValueMovement';
+const cancel_deal_info = 'CancelDealInfo';
+const test_id = 'dt_purchase_test_contract_type_price';
+const localized_basis = getLocalizedBasis();
+const default_mock_props = {
+    basis: 'stake',
+    currency: 'USD',
+    growth_rate: 0.03,
+    has_increased: true,
+    is_loading: false,
+    is_accumulator: false,
+    is_multiplier: false,
+    is_turbos: false,
+    is_vanilla: false,
+    should_fade: true,
+    proposal_info: {
+        id: '129106862',
+        cancellation: { ask_price: 1023, date_expiry: 1907128726 },
+        has_error: false,
+        obj_contract_basis: { text: 'payout' },
+        message: test_message,
+        stake: '10',
+    } as unknown as TProposalTypeInfo,
+    type: 'test_contract_type',
+};
+
+jest.mock('../value-movement', () => jest.fn(() => <div>{value_movement}</div>));
+jest.mock('../cancel-deal-info', () => jest.fn(() => <div>{cancel_deal_info}</div>));
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Popover: jest.fn(props => <div>{props.message}</div>),
+    MobileWrapper: jest.fn(({ children }) => <div>{children}</div>),
```

Comment:
Mocked `MobileWrapper` as it's using deprecated `isMobile`  from `@deriv/shared`;